mediacenter/tests/test_search.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from search.models import Search
+
+from ..models import Document
+
+pytestmark = pytest.mark.django_db
+
+
+def test_document_are_indexed(video):
+    assert Search.objects.count() == 1
+    assert len(list(Search.search(public=True))) == 1
+    assert Document.objects.search("Ikinyugunyugu").count() == 0
```

Comment:
Ok as you want, you're the boss. But I don't agree. It's different, where is the limit of what you know that is already done by the framework and you don't test, and the parts that you test ? I won't argue ;)